gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -253,6 +252,28 @@ public Iterable<String> asArguments() {
         }
     }
 
+    private static List<String> checksToApplySuggestedPatchesFor(
```

Comment:
curious, when does this run? does the output of this feed into up-to-date checks? I'm seeing the argument provider and not sure when things are called etc.

Reply:
It's used in the two of the argument providers above. Basically any time `-PerrorProneApply` is set, but if `-PerrorProneSuppress` is also set we need to do something different, so extracted to a method.

Also, re: up-to-date, these compilations rerun every time basically.